src/test/java/org/jabref/logic/util/strings/StringFormatterTest.java

```diff
@@ -0,0 +1,25 @@
+package org.jabref.logic.util.strings;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class StringFormatterTest {
+
+    @Test
+    void formatErrorMessage() {
+        StringFormatter stringFormatter = new StringFormatter();
+        String error = "The server time zone value 'MSK' is unrecognized or represents more than one time zone. You must configure either the server or JDBC driver (via the serverTimezone configuration property) to use a more specifc time zone value if you want to utilize time zone support.";
+        Exception exceptionUnderTest = new Exception(error);
+        String errorToBeThrown = stringFormatter.formatErrorMessage(exceptionUnderTest);
+        String[] lines = errorToBeThrown.split(System.lineSeparator());
```

Comment:
Please compare it directly to the expected error message (e.g `assertEquals("...", errorToBeThrown)).

Reply:
np

Reply:
@tobiasdiez 

any type of setWidth seems to cut off the string, unless I do it incorrectly

ex,
 exceptionDialog.getDialogPane().setMaxWidth(mainWindow.getWidth() / 3);

<img width="589" alt="image" src="https://user-images.githubusercontent.com/6670698/57927336-f95fef00-78ad-11e9-94b1-4e5c791ab54d.png">

Reply:
Thanks for testing. It seems to be a bug in the library we use. I've proposed a fix at https://github.com/controlsfx/controlsfx/pull/1155. Let's hope they include it quickly. If not, then we can temporarily go with your fix and then at some point fix it in the dialog service class.

Reply:
Cool, thanks

Reply:
It should be fixed now in controlsfx. To see it you need to change 
https://github.com/JabRef/jabref/blob/master/build.gradle#L139
to `controlsfx-8.40.16`.